packages/blade/src/components/Accordion/AccordionItemHeader.tsx

```diff
@@ -56,4 +58,8 @@ const AccordionItemHeader = ({
   );
 };
 
+const AccordionItemHeader = assignWithoutSideEffects(_AccordionItemHeader, {
```

Comment:
do we still need this assignWithoutSideEffects? 🤔

Reply:
We didn't test after the bundling changes we might not need it 🤔